### PR TITLE
Feature/upload profile picture

### PR DESCRIPTION
**Pull request by JorgeMichelena**

### CODE REVIEW

File: api/serializers.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ class Meta:
                   'email',
                   'first_name',
                   'last_name',
-                  'gender')
+                  'gender',
+                  'profile_picture')
```
Comment (MathiasLantean) [inline api/serializers.py:19]: I should end this with a "," so,  if in the future you need to add another field the git history stays clean, just one line changes.

Comment (MathiasLantean) [reply api/serializers.py:19]:
```suggestion
                  'profile_picture',
                 )
```

File: targets/migrations/0010_auto_20200519_0028.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('targets', '0009_auto_20200519_0014'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='topic',
+            name='picture',
+            field=models.ImageField(default='topic-pictures/default-topic.png', upload_to='topic-pictures/'),
```
Comment (MathiasLantean) [inline targets/migrations/0010_auto_20200519_0028.py:16]: Is `default-topic.png` committed? You should add in git those files which are tied to the code.

File: targets/models.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 from django.contrib.gis.db import models as gis_models
 from targets.validators import validate_coordinates
+from django.conf import settings
 
 
 class Topic(models.Model):
     name = models.CharField(max_length=30, unique=True)
-    picture = models.TextField(default='')
+    picture = models.ImageField(default=settings.DEFAULT_TOPIC_PICTURE,
+                                upload_to=settings.TOPIC_PICTURE_FOLDER)
```
Comment (kaozdl) [inline targets/models.py:10]:
```suggestion
    picture = models.ImageField(default=settings.DEFAULT_TOPIC_PICTURE,
                                upload_to=settings.TOPIC_PICTURE_FOLDER,
    )
```

File: users/models.py [hunk 4]
```diff
@@ -15,6 +16,8 @@ class Gender(models.TextChoices):
         max_length=1,
         default='N',
     )
+    profile_picture = models.ImageField(default=settings.DEFAULT_PROFILE_PICTURE,
+                                        upload_to=settings.PROFILE_PICTURE_FOLDER)
```
Comment (kaozdl) [inline users/models.py:20]:
```suggestion
    profile_picture = models.ImageField(default=settings.DEFAULT_PROFILE_PICTURE,
                                        upload_to=settings.PROFILE_PICTURE_FOLDER,
    )
```

**Pull request merged**